lamindb/_artifact.py

```diff
@@ -212,7 +213,11 @@ def hash_size(file):
                 file_size = file.stat().st_size
                 return hash_file(file, file_size)[0], file_size
 
-            n_workers = len(psutil.Process().cpu_affinity())
+            try:
+                n_workers = len(psutil.Process().cpu_affinity())
+            except AttributeError:
+                # Fallback method
+                n_workers = os.cpu_count()
```

Comment:
probably better to use `psutil.cpu_count()` here instead of `os.cpu_count()`

Reply:
Ok!